src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -297,8 +297,30 @@ template <typename TagList, typename Tag>
 using has_subitems = tmpl::not_<
     std::is_same<typename Subitems<TagList, Tag>::type, tmpl::list<>>>;
 
+template <typename Caller, typename Argument, typename IsOk>
```

Comment:
"IsOk" is not the most helpful variable name. Maybe "FoundComputeArgumentInBox"?

Reply:
Since these are tags, maybe `ComputeTag` and `ArgumentTag`?